controllers/validator.go

```diff
@@ -63,32 +60,6 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 		}
 	}
 
```

Comment:
Should we add some validation for new dataMover?

Reply:
probably.

Reply:
There's not really any config for datamover, so I don't know that we have any datamover-specific validation. To enable DataMover, the user just needs to add CSI plugin and enable node agent. Both of these two things can also be used independently from each other for non-datamover cases.

Reply:
Yeah, only thing I thought about was user adding `defaultSnapshotMoveData` in DPA and then we check if both node agent and csi are present

Reply:
@mateusoliveira43 Oh, actually, yes, that would be something to validate. If they set that to true, then it's only useful if node agent (or the deprecated restic config) and CSI plugin are present.

Reply:
we will have a future PR that introduces that to DPA, right? So, can I resolve this conversation and bring it back in that PR?

Reply:
+1 separate add PR.

Reply:
Alright, resolving, thanks!